stratum/hal/lib/tdi/tdi_sde_table_entry.cc

```diff
@@ -171,16 +171,20 @@ ::util::Status TdiSdeWrapper::GetTableEntry(
   RET_CHECK(real_table_data);
   const ::tdi::Table* table;
   RETURN_IF_TDI_ERROR(tdi_info_->tableFromIdGet(table_id, &table));
-
   const ::tdi::Device* device = nullptr;
   ::tdi::DevMgr::getInstance().deviceGet(dev_id, &device);
   std::unique_ptr<::tdi::Target> dev_tgt;
   device->createTarget(&dev_tgt);
 
   const auto flags = ::tdi::Flags(0);
-  RETURN_IF_TDI_ERROR(table->entryGet(*real_session->tdi_session_, *dev_tgt,
-                                      flags, *real_table_key->table_key_,
-                                      real_table_data->table_data_.get()));
+  tdi_status_t status = table->entryGet(*real_session->tdi_session_, *dev_tgt,
```

Comment:
This will return OK if entryGet returns an error other than TDI_TABLE_NOT_FOUND. Is that really what you want?

I would have expected it to be:
```c
  if (status == TDI_TABLE_NOT_FOUND) {
    return MAKE_ERROR(::util::error::Code::NOT_FOUND).without_logging()
           << "No matching table entry";
  }
  return status;
```
If it _is_ your intention, you should add a comment before the `return ::util::okStatus();` explaining why you're ignoring other errors.

Reply:
I checked the SDE code and it returns success or TDI_TABLE_NOT_FOUND.

Reply:
It's hazardous to build the assumption that you know "all the possible error code" into your code. The SDE code might change, and (lest we forget) P4CP supports more than one SDE.